ec/acc/deployment_ccs_test.go

```diff
@@ -63,10 +63,12 @@ func TestAccDeployment_ccs(t *testing.T) {
 					resource.TestCheckResourceAttrSet(ccsResName, "elasticsearch.0.remote_cluster.0.deployment_id"),
 					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.remote_cluster.0.alias", "my_source_ccs"),
 
-					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_data", "true"),
-					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_ingest", "true"),
-					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_master", "true"),
-					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_ml", "false"),
+					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_data", ""),
+					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_ingest", ""),
+					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_master", ""),
+					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.node_type_ml", ""),
+					resource.TestCheckResourceAttr(ccsResName, "elasticsearch.0.topology.0.id", "hot_content"),
+					resource.TestCheckResourceAttrSet(ccsResName, "elasticsearch.0.topology.0.node_roles.#"),
```

Comment:
I thought about it, but `node_roles` are likely to evolve over time, if we're too strict in our assertions we'll end up having a lot of maintenance overhead, so I've opted to at least for now check this lightly rather than assert all the node roles for all topology elements.